presto-local-file/pom.xml

```diff
@@ -105,11 +105,5 @@
             <artifactId>testing</artifactId>
             <scope>test</scope>
         </dependency>
-
```

Comment:
looks like an unrelated change

Reply:
It's needed because this commit removes the only usage of that dependency (`javax.annotation`).